### DISCUSSION TITLE
2 Ideas

### DISCUSSION THREAD

**Discussion by Zoda390** [MaxRobinsonTheGreat/EvolutionSimulatorV2#25] [Ideas and Suggestions]
So imagine adding a new type of wall, one that was breakable by a new cell part (a digger) and could be laid down with a different cell part (a builder), both actions would obviously require some energy. Giving organisms the ability to make shelters from killers, or to evolve some sort of ant like creature.

Also another idea for a cell part, could be a data cell part, that could be read by the eye, as just another thing for the organisms brain to use, as it would produce some value from a range with some color display depending on the value. This could be helpful in evolving more symbiotic or heard type creatures. Maybe have it be a base color of white with a smaller square in the middle with its respective data color.

**Comment by doo158** [comment]
as for the digger cell, i feel like there would be only 2 possible outcomes, depending on the abundance of the diggable walls. if there are very few diggable walls, everything will evolve to live with them and have no diggers. if there are an abundance of them, then everything will evolve with a digger on its face, and completely ignore them. you have to remember this game is not world renowned for generating biodiversity.

secondly, i'm assuming the data cell is meant to mimic other types of cells? if it is, then it would probably be of an extremely niche use, considering that A, creatures have extremely limited senses, and B, you could probably achieve a better effect by just having the actual cell on your body.

**Reply by Zoda390** [reply]
I definitely see how the digger cell would become obsolete due to the sim not having the best biodiversity, I just thought it would be cool if little colonies built walls around themselves to protect themselves and their producers, but that is quite a big thing to ask from random generated cells

But the data cell was meant to stop/replace how the cells would grow eyes or other parts just as a type of identification. It wouldn't mimic any other cell types, I would just be a type of identification the cell colonies could grow to recognize each other. Just like how real life creatures of the same species usually have similar colors or color patterns they use to recognize their own species.

**Reply by doo158** [reply]
oooh i see. yeah, i would imagine if a creature had a couple data cells tacked on its body, it could definately form some cool pack dynamics. would also be helpful to stop all my predators from killing each other the moment i spawn them in lol
